polaris-core/src/main/java/org/apache/polaris/core/persistence/resolver/PolarisResolutionManifest.java

```diff
@@ -383,7 +397,13 @@ public PolarisResolvedPathWrapper getResolvedPath(
     if (resolvedPath == null) {
       return null;
     }
-    if (resolvedPath.getRawLeafEntity() != null
+    // In the case of a passthrough facade, we may have only resolved part of the parent path
+    // in which case the subtype wouldn't match; return the path anyways in this case.
+    //
+    // TODO: Reconcile how we'll handle "TABLE_NOT_FOUND" or "VIEW_NOT_FOUND" semantics
```

Comment:
In what case do we not just forward the return code from the remote catalog?

Reply:
We will. This ResolutionManifest run occurs during the parent-inherited authz phase of request handling prior to forwarding any request to the remote catalog.